packages/app/components/Settings/AddOverrideInput.tsx

```diff
@@ -0,0 +1,57 @@
+import { API } from "@koh/api-client";
+import { Role, UpdateCourseOverrideResponse } from "@koh/common";
+import { Button, Input, message, Select } from "antd";
+import React, { ReactElement, useState } from "react";
+import styled from "styled-components";
+
+const OverrideInput = styled.div`
+  display: flex;
+`;
+
+type AddOverrideInputProps = {
+  courseId: number;
+  onAddOverride: (ovr: UpdateCourseOverrideResponse) => void;
+};
+
+export default function AddOverrideInput({
+  courseId,
+  onAddOverride,
+}: AddOverrideInputProps): ReactElement {
+  const [email, setEmail] = useState("");
+  const [role, setRole] = useState(Role.STUDENT);
+
+  const addOverride = async () => {
+    try {
+      const resp = await API.course.addOverride(courseId, { email, role });
+      onAddOverride(resp);
```

Comment:
Minor UX suggestions
* Add a `message.success` when a new override is created
* Clear the input box if the override was successfully created